custom_components/spook/ectoplasms/cloud/switch.py

```diff
@@ -142,8 +137,8 @@ def is_on(self) -> bool | None:
 
     async def async_turn_on(self, **_kwargs: Any) -> None:
         """Turn the entity on."""
-        await self.entity_description.set_fn(self._cloud, enabled=True)
+        await self.entity_description.set_fn(self._cloud, True)  # noqa: FBT003
```

Comment:
With `enabled` this results in an `call-arg` error. `set_fn` is only typed as a `Callback`. The alternative would be to use a callback protocol which would allow for named arguments.
```
.../cloud/switch.py:140: error: Unexpected keyword argument "enabled"  [call-arg]
```